projects/elife.yaml

```diff
@@ -341,13 +370,6 @@ lax:
             bot-lax-{instance}-inc: {}
             bot-lax-{instance}-out: {}
     aws-alt:
-        standalone:
```

Comment:
if you remove this, all `lax-formula` builds will start to build on 18.04?

Reply:
Same goes for the other `standalone` configurations: they should be pinned to 14.04 now (with the AMI) and that should be removed for each of them when the formula is ready for 18.04 and the instances are recreated.

Reply:
no, you're mistaken. The default `standalone` configuration doesn't specify an ami, just that it should be masterless. The ami is pulled from the project configuration, which is overridden (pinned) to use a deprecated 14.04/16.04. 

you can test this yourself with:

```
source ./venv/bin/activate
./.project.py lax
```

Reply:
Good to go then

Reply:
Some pins were badly specified and `elife-alfred-formula` started building on 18.04: https://github.com/elifesciences/builder/pull/488

Reply:
nuts, I see what you're saying, thanks for the fix.